.vscode/cspell.json

```diff
@@ -271,6 +272,11 @@
     "azurestackhci",
     "backoff",
     "boringssl",
+    "Dgpg",
+    "Dskip",
+    "mvnw",
```

Comment:
I am not sure why we would need these 3 values here. All of them seem like command line tools for building the package. 

May be we do, I am not that familiar with vscode, however, if these were working without these 3 commands, then I would expect it to continue working without the need to add these :)

Reply:
@TheovanKraay  I think we should revert this change, but not sure.

Reply:
One of the checks fails without it... it is using this file to check for spelling errors in markdown files I believe.